llvm/lib/Target/PowerPC/PPCISelLowering.cpp

```diff
@@ -17250,8 +17250,7 @@ bool PPCTargetLowering::getTgtMemIntrinsic(IntrinsicInfo &Info,
 EVT PPCTargetLowering::getOptimalMemOpType(
     const MemOp &Op, const AttributeList &FuncAttributes) const {
   if (getTargetMachine().getOptLevel() != CodeGenOptLevel::None) {
-    // We should use Altivec/VSX loads and stores when available. For unaligned
-    // addresses, unaligned VSX loads are only fast starting with the P8.
```

Comment:
Have you compared the performance w/o unaligned vector-scalar load/store on powr7? The comment looks indicating why it is not performed prior pwr8.